Apim 9518 improve upgrader logs

SamHardi opened the pull request:
## Issue

https://gravitee.atlassian.net/browse/APIM-9518

## Description

Add the exception object into upgrader's error logs when 
missing.
Remove manual logger instantiation.
Simplify the log message in update methods.

## Additional context

A new fix with a breaking change will be added later on master (evolution of gravitee-node).

<!-- Storybook placeholder -->
---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-axgdttqcud.chromatic.com)
<!-- Storybook placeholder end -->

phiz71 commented:
you don't need the apply on 4.4.x label... you are already targeting 4.4.x branch 😉

SamHardi committed:
fix(service): add exception into upgrader's logs

https://gravitee.atlassian.net/browse/APIM-9518
Add the exception object into upgrader's error logs when
missing.
Remove manual logger instantiation.
Simplify the log message in update methode.

wbabyte reviewed (approved):
LGTM 👍

wbabyte reviewed (commented):
when you backport to newer versions make sure you also update upgraders there, since new upgraders might be implemented with the same issue
